app/dashboard/src/components/AriaComponents/Form/components/Reset.tsx

```diff
@@ -42,14 +42,17 @@ export function Reset(props: ResetProps): React.JSX.Element {
     ...buttonProps
   } = props
 
-  const { formState } = formContext.useFormContext(props.form)
+  const form = formContext.useFormContext(props.form)
+  const { formState } = form
 
   return (
     <ariaComponents.Button
+      onPress={() => {
```

Comment:
`type="reset"`doesn't work, right?

Reply:
yup, i had to remove `type="reset"` - it seems like it uses html `reset` behavior which seems to clear the inputs - i should add a comment

Reply:
yeah, ideally, we want to use the native api as much as possible, but it seems like it's not always the case though